confluent_kafka/__init__.py

```diff
@@ -634,3 +634,26 @@ def __repr__(self):
 
     def __str__(self):
         return "{}".format(self.partition)
+
+
+class ThrottleEvent (object):
+    """
+    ThrottleEvent passed as an argument to throttle_cb.
+    Contains details about requests throttled by the broker.
+
+    This class is typically not user instantiated.
+
+    :ivar broker_name: The hostname of the broker which throttled the request
+    :ivar broker_id: The broker's id
+    :ivar throttle_time_ms: The delay associated with the request(seconds)
+    """
+    def __init__(self, broker_name=None,
+                 broker_id=-1,
+                 throttle_time_ms=-1.):
+
+        self.broker_name = broker_name
+        self.broker_id = broker_id
+        self.throttle_time_ms = throttle_time_ms
```

Comment:
Timings are typically defined as float seconds in python, so change this to a double `throttle_time`